app.js

```diff
@@ -0,0 +1,28 @@
+const { checkConnection, getTopics } = require("./controllers/topics.controller.js");
+const express = require("express");
+const app = express();
+app.use(express.json());
+
+// endpoints
+app.get("/api", checkConnection);
+app.get("/api/topics", getTopics);
+
+//error handling
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+});
+
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Bad Request" });
+  } else next(err);
+});
```

Comment:
These error handlers are not relevant to this endpoint, so it is potentially confusing for other developers to see them as part of this pull request. I imagine they may well be useful at some point, but for now, they shouldn't be in the codebase.

Reply:
Removed